server/view/render.js

```diff
@@ -0,0 +1,22 @@
+import {getEnvWithExtensions} from './env-utils';
+
+export default function render(root) {
+  return (ctx, next) => {
```

Comment:
`ctx` is the context used by `Koa`.
It's then passed through all the `use`s, which are generators / async functions.